src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -1594,6 +1594,10 @@ private class RD2_DataMigration_TEST {
         Id jobId = runDataMigration();
 
         List<Error__c> errors = errorGateway.getRecords();
+        if (isDayOfMonthRestrictedPicklistError(errors)) {
```

Comment:
There are two tests fixed in this PR that fail when today is 29th, 30th or 31st, ie when today cannot be translated to the Last_Day since the org does not contain the picklist value.

Reply:
Ah. I see. So, if that error is thrown in the unit test, just ignore it. Makes sense. Thank you.